rfcs/rfc-004-tictactoe-program.md

```diff
@@ -0,0 +1,76 @@
+
+Two players want to play tic-tac-toe with each other on Solana.
+
+The tic-tac-toe program has already been provisioned on the network, and the
+program author has advertised the following information to potential gamers:
+* `tictactoe_publickey` - the program's public key
+* `tictactoe_gamestate_size` - the number of bytes needs to maintain the game state
+
+The game state is a well-documented data structure consisting of:
+- Player 1 public key
```

Comment:
A few lines, below it says "Player 1's". Should add the apostrophes here too.

Reply:
✔️